pkg/store/forward/forward.go

```diff
@@ -26,27 +26,27 @@ const (
 
 var (
 	forwardSamples = prometheus.NewCounter(prometheus.CounterOpts{
-		Name: "telemeter_forward_samples_total",
-		Help: "Total amount of samples successfully forwarded",
-	})
-	forwardErrors = prometheus.NewCounter(prometheus.CounterOpts{
-		Name: "telemeter_forward_request_errors_total",
```

Comment:
Do we have alerts for these metrics? If so we should probably change these as well?

Reply:
I'll investigate. In any case, there are no other references to those metrics in this repo, so we'lll need to make adjustments in either obs/cfg or the deployment repos

Reply:
ack, confirmed we don't have any alerts or recording rules referencing these metrics today. eventually we'll want to add some alerts based on the error rates tho

Reply:
ack